### PR TITLE
feat: allows for passthrough of a default/fallback scalar type

### PR DESCRIPTION
**Pull request by jocisneros**
similar to my [namespace imports pr](https://github.com/Code-Hex/graphql-codegen-typescript-validation-schema/pull/846). working with huge graphs can cause issues with lots of undefined scalar types, resulting in [object Object] being filled in.

This PR allows for a default fallback to still allow the schemas to be built.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -176,6 +176,24 @@ export interface ValidationSchemaPluginConfig extends TypeScriptPluginConfig {
...
+   * ```
+   *
+   * @exampleMarkdown
+   * ```yml
+   * config:
+   *   schema: zod
+   *   defaultScalarSchema: z.unknown()
+   * ```
+   */
+  defaultScalarTypeSchema?: string
```
Comment (Code-Hex) [inline src/config.ts:196]: @jocisneros Could you add to also README?

**Review by Code-Hex — COMMENTED** [review]
@jocisneros almost LGTM

File: src/config.ts (hunk 1 above)
Comment (jocisneros) [reply src/config.ts:196]: done 🫡

**Review by Code-Hex — APPROVED** [review]
Good

**Pull request merged**